### DISCUSSION TITLE
Welcome to meetup-notes Discussions!

### DISCUSSION THREAD

**Discussion by wrightmikea** [Emacs-SF/meetup-notes#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wrightmikea** [comment]
Here is another channel for discussing things related to the Emacs-SF meetings and the related meeting notes (and future meetups)

**Comment by agzam** [comment]
This is awesome. Thank you for enabling this feature!
